BEIMA.Backend/AuthService/IAuthenticationService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using BEIMA.Backend.Models;
+using BEIMA.Backend.MongoService;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BEIMA.Backend.AuthService
+{
+    public interface IAuthenticationService
+    {
+        string CreateToken(User user);
```

Comment:
Interface should still have the class/method headers. You can just copy/paste the ones from AuthenticationService.